pdcupdater/utils.py

```diff
@@ -92,6 +92,14 @@ def ensure_release_exists(pdc, release_id, release):
         release_payload.update(dict(
             active=True,
         ))
+
+        # XXX - Both release_type and type should have the same value, but PDC
+        # doesn't expect that the 'type' value should be present.  If it is
+        # present, we fail with a 400 error.  Here, we delete it until the
+        # pungi/PDC authors tell us otherwise.
+        if 'release_type' in release_payload and 'type' in release_payload:
+            del release_payload['type']
```

Comment:
Ah, it comes from [here](https://kojipkgs.fedoraproject.org/compose/Fedora_Docker-25-20170109.0/compose/metadata/composeinfo.json), `composeinfo['payload']['release']`, but then for reasons I don't remember I add `release_payload['release_type'] = 'ga'`, hardcoded, elsewhere in my code.